WordPress/src/main/java/org/wordpress/android/ui/accounts/NewUserFragment.java

```diff
@@ -488,8 +490,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         termsOfServiceTextView.setOnClickListener(new OnClickListener() {
                                                       @Override
                                                       public void onClick(View v) {
+                                                          Resources res = getResources();
+                                                          Configuration conf = res.getConfiguration();
+                                                          String localString = conf.locale.toString();
+                                                          if (localString.contains("_")) {
+                                                              localString = localString.substring(0, localString.indexOf("_"));
+                                                          }
                                                           ActivityLauncher.openUrlExternal(getContext(),
-                                                                  getString(R.string.wordpresscom_tos_url));
+                                                                  getString(R.string.wordpresscom_tos_url).replace( "en", localString ));
```

Comment:
It's probably harmless, but I'd replace this with a proper `String.format()` call and I'd change `R.string.wordpresscom_tos_url` to `https://%s.wordpress.com/tos`